integration_test/common/common.go

```diff
@@ -29,55 +29,71 @@ type ExpectedMetric struct {
 	// The metric type, for example workload.googleapis.com/apache.current_connections.
 	Type string `yaml:"type"`
```

Comment:
Shall we make this required and also below fields.

Reply:
Good catch, will do.

Reply:
Updated